packages/@vue/cli-plugin-router/README.md

```diff
@@ -0,0 +1,9 @@
+# @vue/cli-plugin-router
+
+> router plugin for vue-cli
+
+## Installing in an Already Created Project
+
+``` sh
+vue add @vue/router
```

Comment:
This seems too verbose. I'm thinking about fixing the `add` command to check for `@vue/cli-plugin-foo` if a shorthand id is provided. I'll do it in a separate PR though.